src/Control/Lens/Plated.hs

```diff
@@ -435,6 +435,9 @@ cosmos = cosmosOf plate
 -- @
 -- 'cosmosOf' :: 'Fold' a a -> 'Fold' a a
 -- @
+
+-- The 'Contravariant' constraint isn't required for the implementation. Since any 'Traversal' produced with 'cosmosOf' is more likely than
```

Comment:
I think we don't need to publicly (in haddocks) document implementation details. The comment can go under the implementation.

Reply:
The empty line above prevents the comment from being included in the haddocks. I can move it anyway, if you prefer.

Reply:
empty line is too subtle, it looks like mistake.

Reply:
Done!